third_party/VENDOR-LICENSE/github.com/hashicorp/go-sockaddr/route_info.go

```diff
@@ -1,5 +1,12 @@
 package sockaddr
 
+import "errors"
```

Comment:
Why is it bringing in go code in the LICENSE directories?

Reply:
I'll check if there is anything broken. But it is weird too.

Reply:
The initial CI build failed expecting https://github.com/sigstore/policy-controller/actions/runs/6888723951/job/18738266746#step:5:141 🤔 ❓ ❔ .

Reply:
I've looked at the content of `third-party` and I found it already contained `go` files in https://github.com/sigstore/policy-controller/blob/main/third_party/VENDOR-LICENSE/github.com/letsencrypt/boulder/core/challenges.go